django-testbed/frontend/src/views/UserListView.js

```diff
@@ -1,20 +1,26 @@
 import React, { useState } from 'react';
 import { Button, Modal } from 'antd';
+import { useViewModelCache } from '@prestojs/viewmodel';
 
 import User from '../models/User';
-import useConnected from '../useConnected';
 import useEndpoint from '../useEndpoint';
 import UserCreateUpdateView from './UserCreateUpdateView';
 
 export default function UserListView() {
     const [selectedId, selectId] = useState();
     const [showCreate, setShowCreate] = useState(false);
-    const { data, error } = useEndpoint(User.endpoints.list, {}, { refreshInterval: 10000 });
+    const { data, error } = useEndpoint(
+        User.endpoints.list,
+        {},
+        {
+            // refreshInterval: 1000
+        }
+    );
     if (error) {
         throw error;
     }
 
-    const records = useConnected(data);
+    const records = useViewModelCache(User, cache => (data ? cache.getList(data) : null));
```

Comment:
This is inconsistent with the previous pattern in `UserCreateUpdateView.js`. Either one seems fine (unless you specifically need to return `null` instead of a falsy value which is what the other one does), but should be consistent

Reply:
Done